scripts/genproto.sh

```diff
@@ -1,2 +1,37 @@
-protoc --go_out=. nflog/nflogpb/nflog.proto
-protoc --go_out=. silence/silencepb/silence.proto
+
+#!/usr/bin/env bash
+#
+# Generate all etcd protobuf bindings.
```

Comment:
The source repo is all Apache 2.0, so I'd just add a one-line note of where it was originally taken from and under what license.